src/components/login/AuthDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import { onAuthStateChanged, signOut } from "firebase/auth";
+import { useEffect, useState } from "react";
+import { auth } from '../../firebase-config';
+
+const AuthDetails = () => {
+    const [authUser, setAuthUser] = useState(null);
+
+    useEffect(() => {
+        const listen = onAuthStateChanged(auth, (user) => {
+            if (user) {
+                setAuthUser
```

Comment:
Looks like you forgot to use this hook and set the user once it comes in.
Update with:
`        setAuthUser(user);`

Reply:
done. I did do that at first. But it was throwing an error. I tried different variations to solve why but in the end i just removed it.